site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.tsx

```diff
@@ -279,6 +297,31 @@ const useExternalAuthVerification = (
   };
 };
 
+function DuplicateWarningMessage() {
+  const [isDismissed, dismiss] = useReducer(() => true, false);
+  const theme = useTheme();
+
+  if (isDismissed) {
+    return null;
+  }
+
+  // Setup looks a little hokey (having an Alert already fully configured to
+  // listen to dismissals, on top of more dismissal state), but relying solely
+  // on the Alert API wouldn't get rid of the div and horizontal margin helper
+  // after the dismiss happens. Not using CSS margins because those can be a
+  // style maintenance nightmare over time
+  return (
+    <div css={{ paddingTop: theme.spacing(6) }}>
+      <Margins size="medium">
+        <Alert severity="warning" dismissible onDismiss={dismiss}>
```

Comment:
I feel like this should be an `"info"`. Nothing actually went wrong, the user isn't doing anything sketchy. It's good context to provide, but I think warnings should result from users doing something worth complaining about, not from following the "blessed path"

Reply:
Yeah, agreed. Will get that fixed up